population/client.go

```diff
@@ -271,6 +279,59 @@ func (c *Client) GetAreas(ctx context.Context, input GetAreasInput) (GetAreasRes
 	return areas, nil
 }
 
+func (c *Client) GetArea(ctx context.Context, input GetAreaInput) (Area, error) {
```

Comment:
Looks like the existing functions follow the pattern of returning fooResponse with embedded foo rather than just raw foo